pkgs/tools/package-management/microdnf/default.nix

```
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, cmake, gettext, libdnf, pkgconfig, glib, libpeas, meson, libsmartcols, pcre, help2man }:
+
+stdenv.mkDerivation rec {
+  pname = "microdnf";
+  version = "3.6.0";
+
+  src = fetchFromGitHub {
+    owner = "rpm-software-management";
+    repo = pname;
+    rev = version;
+    sha256 = "0a7lc3qsnblvznzsz3544l3n84184xi85zf7c3m3jhnmpmxsg39h";
+  };
+
+  nativeBuildInputs = [ pkgconfig pcre glib libpeas cmake gettext libsmartcols help2man libdnf ];
```

```suggestion
  nativeBuildInputs = [ pkg-config pcre glib libpeas cmake gettext libsmartcols help2man libdnf ];
```

You are sure these are not libs required at runtime?

---

According to https://github.com/rpm-software-management/microdnf/blob/master/microdnf.spec it only requires libdnf at runtime. I guess libdnf however should have a few more. Let me move that around.

---

`glib libpeas libsmartcols` should probably go to `buildInputs` too, since it seems to link against them. `pcre` is probably just a false positive from CMake: https://github.com/NixOS/nixpkgs/pull/15046#issuecomment-222263655

---

Ok, done!